stonesoup/simulator/tests/test_groundtruth.py

```diff
@@ -67,3 +68,80 @@ def test_multitarget_ground_truth_simulator(transition_model, timestep):
 
     # Check that ground truth paths vary in length
     assert len({len(path) for path in total_paths}) > 1
+
+
+def test_one_target_ground_truth_simulator_switch(transition_model1,
+                                                  transition_model2,
+                                                  timestep):
+    initial_state = State(np.array([[1]]), timestamp=datetime.datetime.now())
+    model_probs = [[0.5, 0.5], [0.5, 0.5]]
+    simulator = SwitchOneTargetGroundTruthSimulator(
+        transition_models=[transition_model1, transition_model2],
+        model_probs=model_probs,
+        initial_state=initial_state,
+        timestep=timestep)
+
+    for step, (time, groundtruth_paths) in \
+            enumerate(simulator.groundtruth_paths_gen()):
```

Comment:
Is this line consistent with those changes in #137

Reply:
Shouldn't be an issue, but I'll resolve on merging.